testsuite/MDAnalysisTests/guesser/test_default_guesser.py

```diff
@@ -0,0 +1,319 @@
+    assert hasattr(u, 'impropers')
+    assert hasattr(u, 'angles')
+    assert hasattr(u, 'bonds')
+
+
+def test_guess_impropers_with_no_angles():
+    "Test guessing impropers for atoms with no angles "
+    "information without adding bonds or angles to universe"
+    u = mda.Universe(datafiles.two_water_gro)
+    u.guess_TopologyAttrs(to_guess=['impropers'])
+    assert hasattr(u, 'impropers')
+    assert not hasattr(u, 'angles')
+    assert not hasattr(u, 'bonds')
+
+
+def bond_sort(arr):
+    # sort from low to high, also within a tuple
+    # e.g. ([5, 4], [0, 1], [0, 3]) -> ([0, 1], [0, 3], [4, 5])
+    out = []
+    for (i, j) in arr:
+        if i > j:
+            i, j = j, i
+        out.append((i, j))
+    return sorted(out)
+
+
+def test_guess_bonds_water():
+    u = mda.Universe(datafiles.two_water_gro)
+    bonds = bond_sort(DefaultGuesser(
+        None, box=u.dimensions).guess_bonds(u.atoms, u.atoms.positions))
+    assert_equal(bonds, ((0, 1),
+                         (0, 2),
+                         (3, 4),
+                         (3, 5)))
+
+
+def test_guess_bonds_adk():
+    u = mda.Universe(datafiles.PSF, datafiles.DCD)
+    u.guess_TopologyAttrs(force_guess=['types'])
+    guesser = DefaultGuesser(None)
+    bonds = bond_sort(guesser.guess_bonds(u.atoms, u.atoms.positions))
+    assert_equal(np.sort(u.bonds.indices, axis=0),
+                 np.sort(bonds, axis=0))
+
+
+def test_guess_bonds_peptide():
+    u = mda.Universe(datafiles.PSF_NAMD, datafiles.PDB_NAMD)
+    u.guess_TopologyAttrs(force_guess=['types'])
+    guesser = DefaultGuesser(None)
+    bonds = bond_sort(guesser.guess_bonds(u.atoms, u.atoms.positions))
+    assert_equal(np.sort(u.bonds.indices, axis=0),
+                 np.sort(bonds, axis=0))
+
+
+@pytest.mark.parametrize("smi", [
+    "c1ccccc1",
+    "C1=CC=CC=C1",
+    "CCO",
+    "c1ccccc1Cc1ccccc1",
+    "CN1C=NC2=C1C(=O)N(C(=O)N2C)C",
+])
+@requires_rdkit
+def test_guess_aromaticities(smi):
+    mol = Chem.MolFromSmiles(smi)
+    mol = Chem.AddHs(mol)
+    expected = np.array([atom.GetIsAromatic() for atom in mol.GetAtoms()])
+    u = mda.Universe(mol)
+    guesser = DefaultGuesser(None)
+    values = guesser.guess_aromaticities(u.atoms)
+    u.guess_TopologyAttrs(to_guess=['aromaticities'])
+    assert_equal(values, expected)
+    assert_equal(u.atoms.aromaticities, expected)
+
+
+@pytest.mark.parametrize("smi", [
+    "c1ccccc1",
+    "C1=CC=CC=C1",
+    "CCO",
+    "c1ccccc1Cc1ccccc1",
+    "CN1C=NC2=C1C(=O)N(C(=O)N2C)C",
+])
+@requires_rdkit
+def test_guess_gasteiger_charges(smi):
+    mol = Chem.MolFromSmiles(smi)
+    mol = Chem.AddHs(mol)
+    ComputeGasteigerCharges(mol, throwOnParamFailure=True)
+    expected = np.array([atom.GetDoubleProp("_GasteigerCharge")
+                         for atom in mol.GetAtoms()], dtype=np.float32)
+    u = mda.Universe(mol)
+    guesser = DefaultGuesser(None)
+    values = guesser.guess_gasteiger_charges(u.atoms)
+    assert_equal(values, expected)
+
+
```

Comment:
```suggestion

@requires_rdkit
```

A test is failing here because aromaticity guessing also requires RDKit.